### PR TITLE
Martin Yeo comments

### PR DESCRIPTION
**Pull request by trexfeathers**

### CODE REVIEW

**Comment by trexfeathers** [comment]
Cartopy Exercise
[https://github.com/SciTools/courses/blob/master/course_content/cartopy_course/cartopy_intro.ipynb?short_path=0ebf141#L392](url)
This could be a failing on my part, but I made an assumption that the information I needed about pole rotation would be within this notebook, and that I had therefore missed some crucial point. It was 20 minutes before I resolved to look at the cartopy documentation

**Comment by trexfeathers** [comment]
Cube Plotting 5.3.3
[https://github.com/SciTools/courses/blob/master/course_content/iris_course/5.Cube_Plotting.ipynb?short_path=065890a#L355](url)
Found this misleading. A scatter plot of air temperature vs longitude would show every temperature at each longitude, instead the proposed solution just plots a 1D slice. I'm assuming the former is possible but inappropriate at this knowledge level, which is why a 1D slice is shown, but in that case the exercise text could be clearer

**Comment by trexfeathers** [comment]
Advanced Concepts 7.4
[https://github.com/SciTools/courses/blob/master/course_content/iris_course/7.Advanced_Concepts.ipynb?short_path=bdb08ee#L497
](url)
Incomplete sentence

**Comment by trexfeathers** [comment]
Now have a CLA, so this should be able to go through...

File: course_content/iris_course/3.Subcube_Extraction.ipynb [hunk 1]
```diff
@@ -331,7 +331,7 @@
    "source": [
     "Combining multiple constraints can be done in couple of ways. One way involves creating _a single Constraint instance_ with multiple requirements. \n",
     "\n",
-    "For example, using our original CubeList, we extract a cube named `air_potential_temperature` with model level number 10."
```
Comment (pp-mo) [inline course_content/iris_course/3.Subcube_Extraction.ipynb]: In this case, AFAICS it _does_ need to be 'air_potential_temperature'.
This data, as originally loaded, is just (awkwardly) different.

File: course_content/iris_course/3.Subcube_Extraction.ipynb [hunk 2]
```diff
@@ -340,7 +340,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "pot_temperature_constraint = iris.Constraint('air_potential_temperature',\n",
```
Comment (pp-mo) [inline course_content/iris_course/3.Subcube_Extraction.ipynb]: as above :  this *does* need to be ..._potential...

File: course_content/iris_course/3.Subcube_Extraction.ipynb [hunk 3]
```diff
@@ -351,7 +351,7 @@
    "source": [
     "To combine different Constraints we use an `&`.\n",
     "\n",
-    "For example, below we extract the data named `air_potential_temperature`, *and* with model level numbers 4 and 10."
+    "For example, below we extract the data named `air_temperature`, *and* with model level numbers 4 and 10."
```
Comment (pp-mo) [inline course_content/iris_course/3.Subcube_Extraction.ipynb]: Likewise, correct as-is

File: course_content/iris_course/3.Subcube_Extraction.ipynb [hunk 4]
```diff
@@ -360,7 +360,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(cubes.extract('air_potential_temperature' & \n",
+    "print(cubes.extract('air_temperature' & \n",
```
Comment (pp-mo) [inline course_content/iris_course/3.Subcube_Extraction.ipynb]: Likewise, correct as-is

File: course_content/iris_course/3.Subcube_Extraction.ipynb (hunk 1 above)
Comment (trexfeathers) [reply course_content/iris_course/3.Subcube_Extraction.ipynb]: Thanks, have changed back. See [this commit](https://github.com/SciTools/courses/pull/177/commits/bd34c93bfb9945f4d1d90770f6534dcb414645ce)

File: course_content/iris_course/3.Subcube_Extraction.ipynb (hunk 3 above)
Comment (trexfeathers) [reply course_content/iris_course/3.Subcube_Extraction.ipynb]: Thanks, have changed back. See [this commit](https://github.com/SciTools/courses/pull/177/commits/bd34c93bfb9945f4d1d90770f6534dcb414645ce)

File: course_content/iris_course/3.Subcube_Extraction.ipynb (hunk 2 above)
Comment (trexfeathers) [reply course_content/iris_course/3.Subcube_Extraction.ipynb]: Thanks, have changed back. See [this commit](https://github.com/SciTools/courses/pull/177/commits/bd34c93bfb9945f4d1d90770f6534dcb414645ce)

File: course_content/iris_course/3.Subcube_Extraction.ipynb (hunk 4 above)
Comment (trexfeathers) [reply course_content/iris_course/3.Subcube_Extraction.ipynb]: Thanks, have changed back. See [this commit](https://github.com/SciTools/courses/pull/177/commits/bd34c93bfb9945f4d1d90770f6534dcb414645ce)

**Pull request merged**